src/primitives.es6

```diff
@@ -79,7 +88,6 @@ export class ElementType {
   }
 
   fromRefract(dom) {
-    this.element = dom.element;
```

Comment:
Doesn't this break round-trips? For example:

```js
minim.registry.register('foo', StringType);

minim.fromCompactRefract(['foo', {}, {}, 'test']).toCompactRefract();
// => ['string', {}, {}, 'test']
```

This means that I **must** create a subclass for each element now and can't just re-use existing ones with the same behavior.